added basic fonts configuration

ThePeach opened the pull request:
- upgraded gravy to alpha 3
- added base configuration of fonts for gravy
- updated json of fonts atom
- added import for Cabin and Roboto
- updated base HTML definition of fonts

james-nash commented on src/ui-lib/sass/00-settings/typography.scss:
```diff
@@ -1,7 +1,10 @@
 
 $font-size-base: unset;
-$line-height-default: 1.2;
+// TODO remove and use gravy `$base-font-size` and `$base-line-height`
```
The existing variables were just placeholders. Please just go ahead and remove them and reference the Gravy-generated ones directly where needed.

james-nash commented on src/ui-lib/sass/00-settings/gravy.scss:
```diff
@@ -1,2 +1,22 @@
+  )
+);
+
+@include gravy-init;
```
Just an idea, but for developer convenience, it might be good to add a comment here with the URL of any Gravy docs that cover the set of vars/maps/mixins it generates at this point.

ThePeach commented:
Updated to Gravy Beta 1.
Killed the TODOs as suggested, moved the initialisation of gravy in the `heydon.scss` file, and added the "sensible typography defaults" that gravy provides. It makes me think if we should actually move some code output somewhere else.

Haven't added any Gravy documentation link, which might be better placed somewhere else perhaps?

james-nash commented:
I think calling those mixins in `_heydon.scss` is fine for now. If for some reason it later turns out not to be the best home, we can always move them.
